File: backend/src/main/java/com/backend/domain/community/dto/response/CommentWriteResponseDTO.java
package com.backend.domain.community.dto.response;

import com.backend.domain.community.entity.Comment;

import java.time.LocalDateTime;

public record CommentWriteResponseDTO(
        Long commentId,
        Long memberId,
        String comment,
        LocalDateTime createDate
){
    public CommentWriteResponseDTO(Comment comment) {
        this(
                comment.getId(),
                comment.getMemberId(),
                comment.getComment(),
                comment.getCreateDate()
        );
    }
}

File: backend/src/main/java/com/backend/domain/community/dto/response/CommentWriteResponseDTO.kt
package com.backend.domain.community.dto.response

import com.backend.domain.community.entity.Comment
import java.time.LocalDateTime

data class CommentWriteResponseDTO(
    val commentId: Long?,
    val memberId: Long?,
    val comment: String,
    val createDate: LocalDateTime?
) {
    constructor(comment: Comment) : this(
        comment.id,
        comment.memberId,
        comment.comment,
        comment.createDate
    )
}